src/Infrastructure.Dapper/KeyManagement/Repositories/UserSignatureKeyPairRepository.cs

```diff
@@ -48,7 +48,7 @@ await connection.QueryAsync(
                 {
                     Id = CoreHelpers.GenerateComb(),
                     UserId = userId,
-                    SignatureAlgorithm = (byte)signingKeys.SignatureAlgorithm,
+                    SignatureKeyPairAlgorithm = (byte)signingKeys.SignatureAlgorithm,
                     SigningKey = signingKeys.WrappedSigningKey,
                     signingKeys.VerifyingKey,
```

Comment:
Just in case we rename this, could we explicitly provide the name ?
```suggestion
                    VerifyingKey = signingKeys.VerifyingKey,
```